src/components/styles/_stack.less

```diff
@@ -86,7 +103,8 @@
   padding: 10px 0 20px 0;
   height: 90%;
   width: 90%;
-  display: flex;
+  // display: flex;
```

Comment:
Ok all looks great just remove this comment

Reply:
Ok the comment has been removed :)